.gitignore

```diff
@@ -0,0 +1 @@
+*.iml
```

Comment:
Can you please remove this file? Such things should go into global .gitignore file